x-pack/legacy/plugins/apm/server/routes/settings/agent_configuration.ts

```diff
@@ -170,6 +171,9 @@ export const agentConfigurationSearchRoute = createRoute(core => ({
     });
 
     if (!config) {
+      server.log(
```

Comment:
There's a `Logger` available in the request context as well, should we use that?